graphics/zimg/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        sekrit-twc zimg 2.9.3 release-
+
+categories          graphics
+platforms           darwin
+license             WTFPL
+
+description         Scaling, colorspace conversion, and dithering library
+
+long_description    The \"z\" library implements the commonly required image \
+                    processing basics of scaling, colorspace conversion, and \
+                    depth conversion. A simple API enables conversion between \
+                    any supported formats to operate with minimal knowledge \
+                    from the programmer. All library routines were designed \
+                    from the ground-up with correctness, flexibility, and \
+                    thread-safety as first priorities. Allocation, buffering, \
+                    and I/O are cleanly separated from processing, allowing \
+                    the programmer to adapt \"z\" to many scenarios.
+
+checksums           ${distname}${extract.suffix} \
```

Comment:
While it's acceptable to list the distfile name here, we usually don't when there's only one.

Reply:
Fixed